NuRadioMC/SignalProp/analyticraytracing.py

```diff
@@ -197,10 +197,242 @@ def get_y_turn( C_0, x1, n_ice, b, delta_n, z_0):
     """
     c = n_ice ** 2 - C_0 ** -2
     gamma_turn, z_turn = get_turning_point(c, b, z_0, delta_n)
-    C_1 = x1[0] - get_y_with_z_mirror(x1[1], C_0, n_ice, b, delta_n, z_0)[0]
-    y_turn = get_y(gamma_turn[0], C_0, C_1, n_ice, b, z_0)
+    C_1 = x1[0] - get_y_with_z_mirror(x1[1], C_0, n_ice, b, delta_n, z_0)
+    y_turn = get_y(gamma_turn, C_0, C_1, n_ice, b, z_0)
     return y_turn
 
+def get_C_1(x1, C_0, n_ice, b, delta_n, z_0):
+    """
+    calculates constant C_1 for a given C_0 and start point x1
+    """
+    return x1[0] - get_y_with_z_mirror(x1[1], C_0, n_ice, b, delta_n, z_0)
+
```

Comment:
This is not your fault so also not your responsibility, but the analytic raytracing module is full of 'helper' methods/functions which have missing/inaccurate docstrings and serve essentially no use. Maybe this would be a good moment to make them all private, i.e. `_get_C_1` etc?